src/Server/EventBusExplorer.Server.API/Controllers/TopicsController.cs

```diff
@@ -13,11 +13,18 @@ namespace EventBusExplorer.Server.API.Controllers;
 [Produces(MediaTypeNames.Application.Json)]
 public class TopicsController : ControllerBase
 {
-    private readonly IServiceBrokerTopicsService _topicService;
+    private readonly IEventBusManagementService _eventBusManagementService;
```

Comment:
I think we need to talk about the names with use for the concepts involved: are we talking about events or messages?
Probably a call would better fit this need.

Reply:
The suggested change has been moved to a separated issue #34